libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -153,9 +157,29 @@ - (void)userAccountManager:(SFUserAccountManager *)userAccountManager
 }
 
 #pragma mark - Private methods
+- (SFUserAccount *) createInterimTestUser:(SFSDKTestCredentialsData *) credsData  {
+    SFUserAccount *newAcct = [[SFUserAccountManager sharedInstance]createUserAccount];
+    newAcct.accountIdentity.orgId = @"__TESTORGID__";
+    newAcct.accessScopes = [NSSet setWithObjects:@"web", @"api", nil];
```

Comment:
I don't know that this is doing what you expect.  `SFUserAccount.accessScopes` is basically an orphaned property with no bearing on the authentication process.  It should probably be removed.  The scopes that are used for the apps are driven by the `SalesforceSDKManager.authScopes` property.